.github/workflows/task-build-artifacts.yml

```diff
@@ -28,7 +28,6 @@ on:
 env:
   REF: ${{ inputs.sha || inputs.ref || github.sha }}  # Define fallbacks for ref to checkout
   BRANCH: ${{ inputs.ref || github.ref_name }}        # Define "branch" name for pack name (used in `make pack`)
-  BOOST_VERSION: ${{ vars.BOOST_VERSION }}
```

Comment:
Need to remember to remove these (this , `LLVM_VERSION` and `RUNS_ON`) from our `vars` once this is merged